app/models/task.py

```diff
@@ -1,5 +1,32 @@
 from app import db
 
-
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.id'), nullable=True)
+    goal = db.relationship("Goal", back_populates="tasks")
+
+    def to_dict(self):
+        if not self.completed_at:
+            is_complete = False
+        else:
+            is_complete = True
+        return {
+            "id": self.id,
+            "goal_id": self.goal_id,
```

Comment:
The `goal_id` key should not be hard-coded. See comments in tests for more information.